woocommerce-services.php

```diff
@@ -895,6 +895,7 @@ public function attach_hooks() {
 			add_filter( 'woocommerce_admin_reports', array( $this, 'reports_tabs' ) );
 			add_action( 'woocommerce_checkout_order_processed', array( $this, 'track_completed_order' ), 10, 3 );
 			add_action( 'admin_print_footer_scripts', array( $this, 'add_sift_js_tracker' ) );
+			add_action(' current_screen', array( $this, 'edit_orders_page_actions' ) );
```

Comment:
We need to make sure `get_current_screen()` function is loaded.

Reply:
there's a space before the action name:

```suggestion
			add_action( 'current_screen', array( $this, 'edit_orders_page_actions' ) );
```

Reply:
Thanks. 

😭 Soon, https://github.com/Automattic/woocommerce-services/pull/2749 will fix this for me!

Reply:
Fixed here 9a0e90c9